### ISSUE TITLE
Selecting subset of outputs from the previous step by suffix

### ISSUE THREAD

**Issue by jrderuiter** [cgat-developers/ruffus#11]
In my pipeline I have a collate step that produces multiple outputs with different file extensions ('.vcf' and '.txt'). In the following step, I would like to only take the '.vcf' file as a single input for that step. Currently I am only able to get the tuple representing both files as an input. Conversely, selecting files with the '.txt' suffix (in a different step), says that no inputs match the specified suffix. 

It seems that ruffus is currently matching file names on the first file name in a collection of outputs. Is this true? Is there a different way to solve my problem?

To recap, I have the following setup (which I can't get to work):

@collate(step_link, regex(r'([\w-]+).(?:tumor|normal).bam$'), (r'\1.vcf', r'\1.txt'))
def step2....

@transform(step2, suffix('.vcf'), '.keep.vcf')
def step3a...

@transform(step2, suffix('.txt'), '.keep.txt')
def step3b...
